Content.Server/Zombies/ZombifyOnDeathSystem.cs

```diff
@@ -88,6 +89,10 @@ public void ZombifyEntity(EntityUid target)
             //Don't zombfiy zombies
             if (HasComp<ZombieComponent>(target))
                 return;
+            
+            //Don't zombify cyborg
+            if (EntityManager.TryGetComponent(target, out CyborgComponent? cyborgComponent))
```

Comment:
Why not use HasComp<> like in the statement above? TryGetComp is more for checking if it exists because you need to use it (hence the `cyborgComponent` variable).

Reply:
Good question. I just derped I guess, lemme fix that